subprojects/core-api/src/main/java/org/gradle/api/tasks/TaskContainer.java

```diff
@@ -239,7 +239,40 @@
      * @since 4.8
      */
     @Incubating
-    <T extends Task> Provider<T> createLater(String name, Class<T> type, Action<? super T> configurationAction);
+    <T extends Task> Provider<T> createLater(String name, Class<T> type, @Nullable Action<? super T> configurationAction);
```

Comment:
Should this be `@Nullable` in the public API?

Reply:
Yeah, probably not.  I moved it to the internal class.